configs/alcf/job_script.cooley.bash

```diff
@@ -1,27 +1,25 @@
 #!/bin/bash
+#COBALT -t 1:00:00
+#COBALT -n 12
```

Comment:
-n stands for number of nodes, not processors, so I think this should be 1.

Reply:
Oh, good catch.